autopush/web/base.py

```diff
@@ -207,6 +210,13 @@ def _write_response(self, status_code, errno, message=None, error=None,
         if status_code == 410:
             self.set_header("Cache-Control", "max-age=86400")
 
+        if self._handling_message and status_code >= 300:
```

Comment:
Ah, I didn't see any metrics for `autopush.notification.bridge.sent` in datadog. I presume a separate issue will be filed for that?